karma.js

```diff
@@ -16,62 +16,82 @@
 
 'use strict';
 
-var launchers = require('./launchers.json');
+module.exports = function configure(packageJSON, customWebpackConfig) {
 
-var config = {
-  plugins: [
-    require('karma-mocha'),
-    require('karma-phantomjs-launcher'),
-    require('karma-sauce-launcher'),
-    require('karma-browserify')
-  ],
-  basePath: '',
-  frameworks: [ 'browserify', 'mocha' ],
-  files: [
-    'lib/*.coffee',
-    'tests/*.spec.coffee'
-  ],
-  exclude: [],
-  preprocessors: {
-    '**/*.coffee': [ 'browserify' ]
-  },
-  browserify: {
-    debug: true,
-    transform: [ require('coffeeify') ],
-    extensions: [ '.js', '.coffee' ]
-  },
-  reporters: [ 'progress' ],
-  port: 9876,
-  colors: true,
-  autoWatch: false,
-  browsers: [ 'PhantomJS' ],
-  singleRun: true,
-  concurrency: Infinity,
+  var webpackConfig = customWebpackConfig || {
+    module: {
+      rules: [
+        {
+          test: /\.js$/,
+          exclude: /(node_modules)/,
+          loader: "babel-loader",
+          query: {
+            presets: [ "env", "es2015" ]
+          }
```

Comment:
I'm updating the projects to use the **built** files for tests as I think it's the better approach - we're testing what we're publishing.
That  way the tests are in ES2015, and the built files are in ES5 or ES2015 (depending on the project's TS config), TS building is on the project.